components/style/themes/default.less

```diff
@@ -681,3 +681,7 @@
 // Typography
 // ---
 @typography-title-font-weight: 600;
+@typography-title-margin-top: 1.2em;
+@typography-title-margin-right: 0;
```

Comment:
We don't need `@typography-title-margin-right` and `@typography-title-margin-left`, drop them and make it hard coded.

Reply:
Then it will be impossible to use the margin in the theming. Why do you need to hardcode this? My projects require margin.

Reply:
I don't see any neccessary of setting `margin-left` and `margin-right` to headings, could you point out for me by examples?